src/binary/requests/saveSnippet.ts

```diff
@@ -0,0 +1,16 @@
+export type SaveSnippetRequest = {
+  code: string;
+  filename: string;
+  start_offset: number;
+  end_offset: number;
+};
+
+export type SuccessSaveSnippetResponse = "Success";
+
+export type ErrorSaveSnippetResponse = {
+  Error: string;
```

Comment:
Why capital `Error`?

Reply:
This is how the json is being received from the binary (enum variant)